<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn_Page</class>
 <widget class="QWidget" name="SignIn_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignIn_Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="InputNumber">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>129</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputPassWord">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>249</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>72</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>72</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;
color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/new/prefix1/background1.jpg);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 11pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>InputNumber</zorder>
  <zorder>InputPassWord</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
